### DISCUSSION TITLE
Installing fluidsynth returns with "No available formula with the name "fluid-synth""

### DISCUSSION THREAD

**Discussion by Ranger6012** [Homebrew/discussions#3139] [Casks] [answered]
I am attempting to install fluidsynth using Homebrew, however when I use the command found on the Homebrew website
`brew install fluid-synth`
It returns with the following
`Warning: No available formula with the name "fluid-synth".
==> Searching for similarly named formulae...
Error: No similarly named formulae found.
==> Searching for a previously deleted formula (in the last month)...
Error: No previously deleted formula found.
==> Searching taps on GitHub...
Error: No formulae found in taps.
`
The same is true if I try using the command found on the fluidsynth GitHub page.
`brew install fluidsynth`
`Warning: No available formula with the name "fluidsynth".
==> Searching for similarly named formulae...
Error: No similarly named formulae found.
==> Searching for a previously deleted formula (in the last month)...
Error: No previously deleted formula found.
==> Searching taps on GitHub...
Error: No formulae found in taps.
`
Has anyone else had this problem? Is there a command I can use that will work?

**Comment by SMillerDev** [comment, marked as answer]
What do `brew doctor` and `brew config` say?

**Reply by Ranger6012** [reply]
`brew doctor` lead me to the problem and fluid-synth has now installed correctly. Thank you for the lead
